met-api/src/met_api/services/submission_service.py

```diff
@@ -66,18 +74,34 @@ def _validate_fields(submission):
             raise ValueError('Engagement not open to submissions')
 
     @classmethod
-    def review_comment(cls, submission_id, status_id, external_user_id) -> SubmissionSchema:
+    def review_comment(cls, submission_id, reasons: dict, external_user_id) -> SubmissionSchema:
         """Review comment."""
         user = UserService.get_user_by_external_id(external_user_id)
+        status_id = reasons.get('status_id', None)
+        has_personal_info = reasons.get('has_personal_info', None)
+        has_profanity = reasons.get('has_profanity', None)
+        has_threat = reasons.get('has_threat', None)
+        rejected_reason_other = reasons.get('rejected_reason_other', None)
 
         valid_statuses = [status.id for status in CommentStatus.get_comment_statuses()]
 
         if not status_id or status_id == Status.Pending.value or status_id not in valid_statuses or not user:
-            raise ValueError('Invalid review')
+            raise ValueError('Invalid review status.')
+
+        if status_id == Status.Rejected.value and \
+           has_personal_info is not True and \
+           has_profanity is not True and \
+           has_threat is not True and \
```

Comment:
not any () might be useful to write this in a better way ;but good as is as well..

Reply:
updated